understanding/reflow.html

```diff
@@ -209,7 +209,7 @@ <h3>横書き及び縦書きの言語でのスクロールの回避</h3>
                   
                   <h3>リフローと達成基準 1.4.4 テキストのサイズ変更の関係</h3>
                   
-                  <p>リフローの達成基準の焦点は、利用者が 2 方向にスクロールしなくてもズームインできるようにすることである。<a href="resize-text" class="sc">達成基準 1.4.4 テキストのサイズ変更</a>も適用することで、リフローの要件を同時に満たしながら、全てのテキストのサイズを少なくとも 200 ％に拡大することができる。ほとんどの実装では、テキストは拡大率が上がるにつれて拡大し続けることが予想されるため、利用者はテキストを 400 ％まで (そしてそれ以上) に拡大することができる。人々がズームインするときにテキストのサイズが一貫して増加しない実装 (小さな画面の用途に適応するためにメディアクエリに基づいてテキストが変換される場合など) では、テキストのサイズ変更の基準を満たすために、依然として 200％ の拡大することが可能でなければならない。
+                  <p>リフローの達成基準の焦点は、利用者が 2 方向にスクロールすることなしに、ズームインできるようにすることである。<a href="resize-text" class="sc">達成基準 1.4.4 テキストのサイズ変更</a>も適用することで、リフローの要件を同時に満たしながら、全てのテキストのサイズを少なくとも 200 ％に拡大することができる。ほとんどの実装では、テキストは拡大率が上がるにつれて拡大し続けることが予想されるため、利用者はテキストを 400 ％まで (そしてそれ以上) に拡大することができる。人々がズームインするときにテキストのサイズが一貫して増加しない実装 (小さな画面の用途に適応するためにメディアクエリに基づいてテキストが変換される場合など) では、テキストのサイズ変更の基準を満たすために、依然として 200％ の拡大することが可能でなければならない。
```

Comment:
原文は without scrolling ではなく without having to scroll と言っているので、このあたりをもう少ししっかり訳出できると良いのかもしれません。スクロールしないというより、スクロールする必要がない、スクロールが要求されないということだと思います。その観点だと修正前の方が原文のニュアンスに近いような……。

利用者が 2 方向のスクロールを必要とすることなくズームインできる
利用者に 2 方向のスクロールを求めることなく、ズームインできる

というくらいでしょうか。もっと大胆に意訳するなら

利用者がズームインしても、2 方向のスクロールを必要としないようにする

というくらいにしてしまうのもありでしょうか。意訳しすぎかもしれませんが。

issue で言われている「2方向でスクロールすることでズームインができるが、そのようなズームインをしないようにするというように読めます」については、 *原文もそう読める* のでそんなに気にしなくて良いとは思っています。

……個人的にはむしろ後半の「テキストのサイズが一貫して増加しない」がヤバイと思っていて、「増加しない」と読めてしまいほぼ逆の意味にとれてしまいます。ので、こっちを直したいです。こまごま直してこんな感じでどうでしょうか?

```suggestion
                  <p>リフローの達成基準の焦点は、利用者がズームインしても、2 方向のスクロールを必要としないようにすることである。<a href="resize-text" class="sc">達成基準 1.4.4 テキストのサイズ変更</a>も適用することで、リフローの要件を同時に満たしながら、全てのテキストのサイズを少なくとも 200 ％に拡大することができる。ほとんどの実装では、テキストは拡大率が上がるにつれて拡大し続けることが予想されるため、利用者はテキストを 400 ％まで (そしてそれ以上に) 拡大することができる。人々がズームインするときにテキストのサイズが一貫して増加するわけではない実装 (小さな画面の用途に適応するために、メディアクエリに基づいて変化する場合など) では、テキストのサイズ変更の基準を満たすために、依然として 200％ の拡大が可能でなければならない。
```